### ISSUE TITLE
[Bug]: Data - Collecting Event, List, Displaying collecting events – Bugs

### ISSUE THREAD

**Issue by michesposito** [SpeciesFileGroup/taxonworks#3224]
### Steps to reproduce the bug

```bash
1. Data
2. Collecting Events
3. List

Geographic areas data is missing/blank in the CE list 

4. double click on verbatim locality
 
Geographic area missing in attributes panel
```

### Screenshot

![data CE display](https://user-images.githubusercontent.com/74991451/207114671-3410bcb6-6a24-4d6d-8095-27edebbfbfee.jpg)

![CE attributes](https://user-images.githubusercontent.com/74991451/207114973-54b5fb43-c5c4-4a7b-bd42-6b59d9904e06.jpg)

### Environment

Sandbox

### Version

build 64eba3530 on 2022-12-12T15:46:38Z

### Browser Used

Chrome Version 108.0.5359.96 Windows 10 Enterprise 21H2

**Comment by michesposito** [comment]
@bpescador, @cgrinter, @rich-keller

**Comment by mjy** [comment]
@AntWeb-org I think this is because the data is created on import, not because it's there and not shown?  

@LocoDelAssembly this came up yesterday in another topic- do we try to link to GeographicArea on DwC Importer?

**Comment by AntWeb-org** [comment]
I was wondering if it had to do with the three types of results for one country, e.g., when filtering geographic area, 'Madagascar' displays:

1. madagascar - country - has shape - 6 users
2. Africa: Western Indian Ocean: Madagascar - TDWG level 3 - has shape
3. Western Indian Ocean: Madagascar; madagascar - TDWG level 4 - has shape

Our data may be #1. I have UI Issue regarding the multiple options that could potentially exclude data.

**Comment by mjy** [comment]
@AntWeb-org excellent, thanks for the example.  I'll poke at it specifically.

**Comment by mjy** [comment]
@AntWeb-org can you let me know the project name, this is on Sandwich?

**Comment by AntWeb-org** [comment]
https://sandwich.taxonworks.org/tasks/collecting_events/filter
but SANDBOX - build [64eba3530](https://github.com/SpeciesFileGroup/taxonworks/tree/64eba3530) on 2022-12-12T15:46:38Z
Thanks!

**Comment by mjy** [comment]
This seems to be a data issue.

This particular record has no GeographicArea. If a GeographicArea is assumed to be imported because others have been then this is an issue with the DwC Importer or your initial data. The DwC inference that fills the filter result uses the Georeference.  See below. 

`/tasks/collecting_events/filter?match_identifiers=PSW10340&match_identifiers_type=identifier`
`/tasks/collecting_events/new_collecting_event?collecting_event_id=756092`

**Comment by rich-keller** [comment]
I agree there is a data issue. When I looked at this a few months back in different context, I noted:

Overall, when I look at the imported records that Dash created, I see that he didn't fill in the field "Geographic_area_id", which references the row in the data table. Instead, he filled in the text names (rather than the record number) of the Country, ADM1, and ADM2. I think this explains the reason why when you search for collecting events with a Geographic_area of "Madagascar" (country), you get only a handful of results. **Most of the Madagascar records are not coded with the Geographic_area ID. Only the records entered by hand (non-imported) have a Geographic_area ID.**

**Comment by LocoDelAssembly** [comment]
> @LocoDelAssembly this came up yesterday in another topic- do we try to link to GeographicArea on DwC Importer?

At present no, but it is possible to import the relevant dwc terms as data attributes if them are set in the project (without resorting to `TW:DataAttribute:...`).

**Comment by mjy** [comment]
Note that we also discovered Madagascar related shape (polygon) issue.

**Comment by AntWeb-org** [comment]
import GA issue
